dbaas-lib/engines/k8s-psmdb/psmdb.go

```diff
@@ -78,15 +78,17 @@ const (
 )
 
 type PSMDBClusterStatus struct {
-	Messages []string `json:"message,omitempty"`
-	Status   AppState `json:"state,omitempty"`
+	Messages []string             `json:"message,omitempty"`
+	Status   AppState             `json:"state,omitempty"`
+	Replsets map[string]AppStatus `json:"replsets"`
 }
 
 type AppStatus struct {
-	Size    int32    `json:"size,omitempty"`
-	Ready   int32    `json:"ready,omitempty"`
-	Status  AppState `json:"status,omitempty"`
-	Message string   `json:"message,omitempty"`
+	Size        int32    `json:"size,omitempty"`
+	Ready       int32    `json:"ready,omitempty"`
+	Status      AppState `json:"status,omitempty"`
+	Message     string   `json:"message,omitempty"`
+	Initialized bool     `json:"initialized"`
 }
```

Comment:
Why you don't use cluster status types from PSMDB?

Reply:
because there are different versions of operator

Reply:
But we're already using different types from different versions for spec: https://github.com/Percona-Lab/percona-dbaas-cli/blob/347b6b05a8963444a85d70517389d4bdffe70287/dbaas-lib/engines/k8s-psmdb/psmdb.go#L8-L10. Why not to do the same with status?

Reply:
done in https://github.com/Percona-Lab/percona-dbaas-cli/pull/101/commits/7ce73ab37cf608c44c28315bf025ea3a3746a2e1